packages/web/src/shared/Footer.3.tsx

```diff
@@ -127,6 +127,8 @@ const Navigation = React.memo(function _Navigation({
   )
 })
 
+const YEAR = new Date().getFullYear()
```

Comment:
why the UPPERCASE here?

Reply:
often when i declare a constant outside of a function I do so in upper case, a habit leftover from the days of ruby, but i think it's common in JS land too